components/Form.tsx

```diff
@@ -46,6 +46,9 @@ export default function Form({ title, questions, listingId, includeEventsAttende
   const [formData, setFormData] = useState<FormData>(initialValues);
   const [resumeFileName, setResumeFileName] = useState<String>("");
   const [imageFileName, setImageFileName] = useState<String>("");
+  const [resumeFileSize, setResumeFileSize] = useState<number>(0);
+  const [imageFileSize, setImageFileSize] = useState<number>(0);
+  const MAX_FILE_SIZE = 6
```

Comment:
What do you think about renaming `MAX_FILE_SIZE` to `MAX_FILE_SIZE_BYTES` and use the direct bytes to represent the value like `6 * 1024 * 1024 - 1` to represent 6 bytes * 1024 --> kb * 1024 --> mb minus one to handle file limit?